source/aggregation-pipeline-builder.txt

```diff
@@ -92,6 +80,14 @@ Open a Saved Pipeline
 
 #. In the modal, click :guilabel:`Open Pipeline`.
 
+.. note::
+
+   Starting in |compass| 1.31, you can view your favorite and saved 
+   queries and aggregation pipelines on the :guilabel:`My Queries` view 
```

Comment:
Maybe omit "favorite" here and just say "saved queries and aggregation pipelines". I don't think users can "favorite" a pipeline so just generalizing to "saved queries and pipelines" is fine here.